prime/script/start.sh

```diff
@@ -5,5 +5,5 @@ exec java \
     -Dfile.encoding=UTF-8 \
     --add-opens java.base/java.lang=ALL-UNNAMED \
     --add-opens java.base/java.io=ALL-UNNAMED \
-    -agentpath:/opt/cprof/profiler_java_agent.so=-cprof_service=prime,-cprof_service_version=1.60.2,-logtostderr,-minloglevel=2,-cprof_enable_heap_sampling \
+    -agentpath:/opt/cprof/profiler_java_agent.so=-cprof_service=prime,-cprof_service_version=1.61.0,-logtostderr,-minloglevel=2,-cprof_enable_heap_sampling \
```

Comment:
The versioning is managed with magic strings, and even repeated. In this case there is a version set in prime/build.gradle, and it would be good if it was declared only once.